Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableEntitiesManager.cpp

```diff
@@ -609,6 +622,31 @@ namespace AzFramework
         }
     }
 
+    bool SpawnableEntitiesManager::ProcessRequest(ClaimEntityCommand& request)
+    {
+        Ticket* ticket = request.m_ticket;
+        if (request.m_requestId == ticket->m_currentRequestId)
+        {
+            AZStd::vector<AZ::Entity*>& spawnedEntities = ticket->m_spawnedEntities;
+            AZStd::vector<AZ::Entity*>::size_type entityIndex = 0;
+            for (entityIndex = 0; entityIndex < spawnedEntities.size(); entityIndex++)
+            {
+                if (spawnedEntities[entityIndex]->GetId() == request.m_entityId)
+                {
+                    spawnedEntities.erase(spawnedEntities.begin() + entityIndex);
+                    ticket->m_spawnedEntityIndices.erase(ticket->m_spawnedEntityIndices.begin() + entityIndex);
+                    ticket->m_currentRequestId++;
+                    return true;
+                }
+            }
+            return false; 
```

Comment:
That's correct. If for one reason or another a command can't be completed that bit of code queues it in a pending queue so it can be tried again later on.